classes/fields/pick.php

```diff
@@ -1304,28 +1304,41 @@ public function build_dfv_field_item_data_recurse_item( $item_id, $item_title, $
 			$edit_link = add_query_arg( array( 'pods_modal' => '1' ), $edit_link );
 		}
 
-		// Determine if this is a selected item
+		// Determine if this is a selected item.
+		// Issue history for setting selected: #4753, #4892, #5014
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: a47f55de534fcce5fe0d9ad2ce17205b</sub>

Reply:
```suggestion
		// Issue history for setting selected: #4753, #4892, #5014.
```